doc/source/reference/arrays.classes.rst

```diff
@@ -39,74 +39,70 @@ Special attributes and methods
 
 NumPy provides several hooks that classes can customize:
 
-.. method:: class.__array_ufunc__(ufunc, method, inputs, **kwargs)
+.. method:: class.__array_ufunc__(ufunc, method, *inputs, **kwargs)
 
-   .. versionadded:: 1.11
+   .. versionadded:: 1.13
 
-   Any class (ndarray subclass or not) can define this method to
-   override behavior of NumPy's ufuncs. This works quite similarly to
-   Python's ``__mul__`` and other binary operation routines.
+   Any class (ndarray subclass or not) can define this method (or set it to
```

Comment:
Commas, not "(ndarray....)". I'd also remove the parenthesis around "or set...".